src/libraries/Microsoft.NETCore.Platforms/src/Microsoft.NETCore.Platforms.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.SDK">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think it would have made more sense for SDK resolver to keep names case insensitive (i doubt it would break anything since changing just the casing and publishing a different SDK to nuget.org doesn't work anyway), and especially since there are legit SDKs available with capital casing, e.g. `Microsoft.NET.SDK.WorkloadAutoImportPropsLocator`, `Microsoft.NET.SDK.WorkloadManifestTargetsLocator`; it's very easy to make this mistake. 😅

Reply:
Agreed...